### PR TITLE
Chair racers gone wild

### PR DESCRIPTION
**Pull request by kentrv**
Implemented some kind of a "solution" to record the wheel speeds.
Also added an exit "menu" by pressing esc.
Extraordinary work accomplished by an amazing individual: me

### CODE REVIEW

File: VR-wheelchair-game/LogFiles_ExciteOMeter/logDebug.txt [hunk 1]
```diff
@@ -538,3 +538,28 @@ datetime,Event
 231102093751.457,New App Execution
 datetime,Event
 231102094240.971,New App Execution
+datetime,Event
```
Comment (anines) [inline VR-wheelchair-game/LogFiles_ExciteOMeter/logDebug.txt]: This file should be added to the .gitignore file

File: WheelchairExe/LogFiles_ExciteOMeter/logDebug.txt [hunk 2]
```diff
@@ -304,3 +304,51 @@ datetime,Event
 231102095834.949,ExciteOMeter >> No incoming data RRInterval was found to calculate feature SDNN
 231102095844.958,ExciteOMeter >> No incoming data RRInterval was found to calculate feature SDNN
 231102095854.966,ExciteOMeter >> No incoming data RRInterval was found to calculate feature SDNN
+datetime,Event
```
Comment (anines) [inline WheelchairExe/LogFiles_ExciteOMeter/logDebug.txt]: Add this file to .gitignore

**Review by anines — CHANGES_REQUESTED** [review]
Please add the tho commented files to .gitignore, then I will gladly approve:D

File: WheelchairExe/LogFiles_ExciteOMeter/logDebug.txt (hunk 2 above)
Comment (kentrv) [reply WheelchairExe/LogFiles_ExciteOMeter/logDebug.txt]: tull og tøys

**Review by WilliamWatcher — COMMENTED** [review]
Great! The more gitignore files the better

**Pull request merged**